client/locales/de/translation.json

```diff
@@ -96,5 +93,12 @@
     "dictCancelUpload": "Hochladen abbrechen",
     "dictRemoveFile": "Datei entfernen",
     "dictMaxFilesExceeded": "Sie können keine weiteren Dateien mehr hochladen"
+  },
+  "preview": {
+    "small": "Mobile",
```

Comment:
As the sizes are now configurable I'd love to see the translations configurable as well. these strings are very specific for the NZZ usecase.
It would make a lot of sense to have these configurable as well.
This could be done here https://github.com/nzzdev/Q-server/blob/dev/plugins/core/editor/routes/locales.js like for the tool names. The `previewSizes` config could hold a `label_locales` property with the translations. The endpoint can then include a `preview` key computed from the `previewSizes` options if given.

I'd still keep the defaults here but name them something like `default__small` with a more generic value and use them in the default `sizeOptions` in `item-preview.js`.

Reply:
Editor translation of q-server are always under the namespace `tools` -> https://github.com/nzzdev/Q-editor/blob/dev/client/src/main.js#L86
But for the preview I need them to be in the default namespace `translation`. I am not sure what the best solution for this could be.

Reply:
I propose that we remove the translation namespace `tools` and let the tool names and preview labels be part of the default namspace `translation`.  We just have to be aware that the default namespace `translation` contains translations defined in q-editor and q-server-nzz. @benib what do you think about this?

Reply:
@manuelroth can you elaborate the benefits of this?